.github/workflows/test_and_deploy.yml

```diff
@@ -3,24 +3,7 @@ name: tests
 on: [push, pull_request]
 
 jobs:
```

Comment:
How does it work now the lint checks are in a different file? Can we still require the lint checks before running the tests? I want to make sure we don't run the lengthy tests if the linting fails, so we don't release the poorly formatted code.

Reply:
interesting question, let me look into that

Reply:
I've got this solved, but for now due to #31 the next step doesn't run (as expected)

Reply:
Just trying to get my head around this, where do we specify _which_ other workflow has to pass before running this one?

Reply:
so I've had to abandon this as it doesn't work as I hoped, but essentially:

In this example `Run Tests` would be the name you specify at the top of the CI file
![image](https://user-images.githubusercontent.com/15052188/154995292-f33aebad-5574-4e82-9eb5-1ac2358a1dbb.png)

then here (a different worfklow called `Build`) we're now checking to see if it was successful or not. I don't know why that isn't a standard option in the first bit, I imagine it would be at a later date
![image](https://user-images.githubusercontent.com/15052188/154995384-bfdbd3b3-24b2-4c96-9e3c-6b078af1c55f.png)